tests/spread/plugins/craft-parts/build-and-run-hello/python-hello/snap/snapcraft.yaml

```diff
@@ -0,0 +1,16 @@
+name: python-hello
+version: "1.0"
+summary: simple python application
+description: build a python application using core22
+base: core22
+confinement: strict
+
+apps:
+  python-hello:
+    command: bin/hello
+    extensions: [gnome]
```

Comment:
Do you intend to include the gnome extension here, since it's included below in the gnome-extension tests?